micro-rdk/src/common/board.rs

```diff
@@ -130,6 +132,10 @@ impl FakeBoard {
     }
 
     pub(crate) fn from_config(cfg: ConfigType) -> Result<BoardType, BoardError> {
+        if cfg.get_attribute::<bool>("fail_new").unwrap_or(false) {
```

Comment:
Is there a test that was supposed to leverage this? Or was it just for hand testing? Do you want to keep it in? Maybe we should reserve a name in the component namespace for an invalid component (`rdk:builtin:invalid`) which must never be registered?

Reply:
This was just for hand testing. The reason I opted for this instead of a new model was because this attribute and functionality already exists for the fake board in RDK and I thought we would benefit from the consistency

Reply:
OK if you are doing as macro-RDK does then good by me. I agree we should be consistent.